### ISSUE TITLE
DSMR & Stream-server don't play nice together 

### ISSUE THREAD

**Issue by hmmbob** [oxan/esphome-stream-server#33]
I was running into a similar issue as #11. Is there already a fix or feasible workaround available for this issue?
Asking because we are slightly past the 6+ months 😉 mentioned there and maybe there is a workaround that I haven't found yet.

The use case is that I have a "SlimmeLezer" hooked up to my meter. I'd like this to do two things:

Send accurate L1/L2/L3 readings to my SmartEVSE (car charger) directly from the SlimmeLezer, for which I think I need the DSMR component in ESPHome. Sending the info is done with a HTTP API call, and I prefer to send it directly from the SlimmeLezer as this is the most stable route (less links in the chain to get broken). Accurate L1/L2/L3 info is needed for Load Balancing.
Send all P1 readings to my DSMR-Reader docker instance on a different host, to keep track of usage.
Right now, when enabling dsmr and stream-server together, I get a lot of CRC errors. Enabling just one of the two, all is fine.

**Comment by oxan** [comment]
> Asking because we are slightly past the 6+ months

Yeah, the changes I had in mind turned out not to be as simple as I thought they'd be. While I can see the use for this feature, I wouldn't expect the situation here to change anytime soon. It's not something I would use personally and requires some fairly involved changes in how UARTs are handled in ESPHome itself.
